testsuite/cluster-test/src/instance.rs

```diff
@@ -69,6 +70,23 @@ impl Instance {
         Ok(())
     }
 
+    pub fn counter(&self, counter: &str) -> Result<f64> {
+        let response: Value =
+            reqwest::get(format!("http://{}:9101/counters", self.ip).as_str())?.json()?;
+        if let Value::Number(ref response) = response[counter] {
+            if let Some(response) = response.as_f64() {
+                return Ok(response);
+            }
+        }
+        Err(format_err!("Could not find counter: {}", counter))
```

Comment:
It would be good to include response here to see what was wrong

Reply:
Done